packages/firestore/src/remote/remote_store.ts

```diff
@@ -814,10 +812,15 @@ export class RemoteStore {
   handleUserChange(user: User): Promise<void> {
     log.debug(LOG_TAG, 'RemoteStore changing users: uid=', user.uid);
 
-    // Tear down and re-create our network streams. This will ensure we get a fresh auth token
-    // for the new user and re-fill the write pipeline with new mutations from the LocalStore
-    // (since mutations are per-user).
-    this.disableNetworkInternal(OnlineState.Unknown);
-    return this.enableNetwork();
+    // If the network has been explicitly disabled, make sure we don't
+    // accidentally re-enabled it.
```

Comment:
"re-enabled" => "re-enable" (sorry if I had that typo in my original suggestion)

Reply:
Done.